gesturelibrary/include/double.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "gesturelib.h"
+#include "math.h"
+#include "recognizer.h"
+#include "stroke.h"
+#include "tap.h"
+
+/// @brief
+typedef struct double_tap {
+    state_t state;
+    int x;
+    int y;
+    int t;
+} double_tap_t;  // single finger double tap data
+
+typedef struct coords {
```

Comment:
If it doesn't need to be exposed to the user, please declare it in the .c file instead.